hazelcast/src/main/java/com/hazelcast/map/impl/query/AggregationResult.java

```diff
@@ -73,6 +81,27 @@ public void onCombineFinished() {
         }
     }
 
+    @Override
+    public void add(QueryableEntry entry) {
+        aggregator.accumulate(entry);
+    }
+
+    @Override
+    public AggregationResult createSubResult() {
+        Aggregator aggregatorClone = serializationService.toObject(serializationService.toData(aggregator));
+        return new AggregationResult(aggregatorClone, serializationService);
+    }
+
+    @Override
+    public void orderAndLimit(PagingPredicate pagingPredicate, Map.Entry<Integer, Map.Entry> nearestAnchorEntry) {
+        // do nothing
```

Comment:
maybe an explanation can be added to say why we are doing nothing in this method to increase readability.